### DISCUSSION TITLE
[Feature] Better scroll button on Android

### DISCUSSION THREAD

**Discussion by Acidiphilium** [immich-app/immich#12507] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm not sure it's a feature or a bug but on my Android 14 (Oneplus 12), I cannot see or let's say, slide easily to years before 2012. I have around 20k pictures from 2002 to today and the blue sliders gets hidden below the bottom bar when I reach pictures around 2011 or below. The slider disappear but I still can slide further if I hold the slider button, though it's hard to get easily to the very last picture from year 2002 and then slider is hidden and only way to go up it's by normal painful scrolling again and again and again till the slider appear again from behind the bottom bar.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
This is a bug 😅 #12188

**Reply by Acidiphilium** [reply]
On yeah sorry for the duplication then :-)
